engine/src/main/java/org/destinationsol/game/ship/ShipBuilder.java

```diff
@@ -274,8 +274,9 @@ private Hull buildHull(SolGame game, Vector2 position, Vector2 velocity, float a
             doors.add(door);
         }
 
+        Fixture base = getBase(hullConfig.hasBase(), body);
```

Comment:
This line is causing the jenkins failure, it must have snuck in during the merge :/